test/Microsoft.AspNetCore.Blazor.E2ETest/Infrastructure/BrowserFixture.cs

```diff
@@ -16,7 +16,18 @@ public BrowserFixture()
         {
             var opts = new ChromeOptions();
             opts.AddArgument("--headless");
-            Browser = new RemoteWebDriver(opts);
+
+            try
+            {
+                Browser = new RemoteWebDriver(opts);
+            }
+            catch (WebDriverException ex)
+            {
+                var message =
+                    "Failed to connect to the web driver. Please see the readme and follow the instructions to install selenium." +
+                    "Remember to start the web driver with `selenium-standalone start` before running the end-to-end tests.";
+                throw new InvalidOperationException(message, ex);
```

Comment:
This is better than the uninformative error we had before.  Later on we could look into launching and stopping selenium server automatically from the tests. This is fine for now though :)